drivers/trone/TROne.hpp

```diff
@@ -0,0 +1,106 @@
+/****************************************************************************
+*
+*   Copyright (C) 2016 PX4 Development Team. All rights reserved.
+*
+* Redistribution and use in source and binary forms, with or without
+* modification, are permitted provided that the following conditions
+* are met:
+*
+* 1. Redistributions of source code must retain the above copyright
+*    notice, this list of conditions and the following disclaimer.
+* 2. Redistributions in binary form must reproduce the above copyright
+*    notice, this list of conditions and the following disclaimer in
+*    the documentation and/or other materials provided with the
+*    distribution.
+* 3. Neither the name PX4 nor the names of its contributors may be
+*    used to endorse or promote products derived from this software
+*    without specific prior written permission.
+*
+* THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+* "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+* LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+* FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+* COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+* INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+* BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+* OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+* AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+* LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+* ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+* POSSIBILITY OF SUCH DAMAGE.
+*
+****************************************************************************/
+
+#pragma once
+
+#include "I2CDevObj.hpp"
+
+namespace DriverFramework
+{
+	/* Configuration Constants */
+
+	#define TRONE_DEVICE_PATH   "/dev/iic-2"  // J15
+	#define TRONE_CLASS_PATH  	"/dev/trone"
+
+	/* TRONE Registers addresses */
+	#define TRONE_MEASURE_REG	0x00		/* Measure range register */
+	#define TRONE_WHO_AM_I_REG  0x01        /* Who am I test register */
+	#define TRONE_WHO_AM_I_REG_VAL 0xA1
+
+	/* Device limits in m*/
+	#define TRONE_MIN_DISTANCE (0.20f)
+	#define TRONE_MAX_DISTANCE (14.00f)
+
+	#define TRONE_CONVERSION_INTERVAL 50000 /* 50ms */
+
+	// TODO: no idea what to use here
+	#define TRONE_MEASURE_INTERVAL_US (1000000/150)
+
+	#define TRONE_SLAVE_ADDRESS		(0x30)
```

Comment:
All these defines are not really inside the namespace. So the indenting is rather confusing. For my taste I would put the outside and non-indented. Or otherwise you could use:
`static const uint8_t TRONE_SLAVE_ADDRESS = 0x30;`